cmd/sys-mgmt-executor/main.go

```diff
@@ -167,31 +162,31 @@ func runDockerCommands(service string, dockerService string, operation string) e
 
 		out, err := cmd.CombinedOutput()
 		if err != nil {
-			agent.LoggingClient.Error(err.Error())
-			agent.LoggingClient.Info("docker command failed: %s", string(out))
+			LoggingClient.Error(err.Error())
+			LoggingClient.Info("docker command failed: %s", string(out))
 		}
 
 		switch operation {
 		case START:
 		case RESTART:
 			if !findDockerContainerStatus(service, "Up") {
-				agent.LoggingClient.Warn("docker start operation failed for service %s", service)
+				LoggingClient.Warn("docker start operation failed for service %s", service)
 			}
 			break
 
 		case STOP:
 			if !findDockerContainerStatus(service, "Exited") {
-				agent.LoggingClient.Warn("docker stop operation failed for service %s", service)
+				LoggingClient.Warn("docker stop operation failed for service %s", service)
 			}
 			break
 
 		default:
-			agent.LoggingClient.Warn("unknown operation %s was requested", operation)
+			LoggingClient.Warn("unknown operation %s was requested", operation)
 			break
 		}
 	} else {
 		err := errors.New("an unknown operation")
-		agent.LoggingClient.Error(fmt.Sprintf("system management was requested to perform %v %s on the %s service", err.Error(), operation, service))
+		LoggingClient.Error(fmt.Sprintf("system management was requested to perform %v %s on the %s service", err.Error(), operation, service))
```

Comment:
same comment as above here, I think it would be better to use `fmt.Errorf` to create the full `error` object that contains the service name, then log that with `err.Error()` using `LoggingClient.Error` and then finally return the error.